.vib/grafana-mimir/goss/goss.yaml

```diff
@@ -0,0 +1,50 @@
+file:
+  {{ .Vars.mimir.dataDir }}/conf/mimir.yaml:
+    mode: "0644"
+    filetype: file
+    exists: true
+    owner: root
+    contains:
+    - "memcachedchunks:{{ .Vars.memcachedchunks.service.ports.memcached }}"
+    - "memcachedfrontend:{{ .Vars.memcachedfrontend.service.ports.memcached }}"
+    - "memcachedindex:{{ .Vars.memcachedindex.service.ports.memcached }}"
+    - "http_listen_port: {{ .Vars.mimir.containerPorts.http }}"
+http:
+  # As we need to make several API requests, we'll use different services (and pods)
+  # to at the same time test most of them are working properly. For those services not used here,
+  # we suppose they are running as expected given that all follow the same arch
+  http://grafana-mimir-query-frontend:{{ .Vars.queryFrontend.service.ports.http }}/services:
+    status: 200
+    body:
+    - "!Stopping"
+    - "!Terminated"
+    - "!Failed"
+  http://grafana-mimir-distributor:{{ .Vars.distributor.service.ports.http }}/services:
+    status: 200
+    body:
+    - "!Stopping"
+    - "!Terminated"
+    - "!Failed"
+  http://grafana-mimir-store-gateway:{{ .Vars.storeGateway.service.ports.http }}/config:
+    status: 200
+    body:
+    - /advertise_port.*{{ .Vars.mimir.containerPorts.gossipRing }}/
```

Comment:
I see no reference to this in the configuration in values.yaml. Maybe it is worth adding it so you can configure the gossip ring port

Reply:
Now that it is configurable, ok!